virtualization/xe-guest-utilities/xe-guest-utilities.yaml

```diff
@@ -0,0 +1,17 @@
+name: xe-guest-utilities
+container:
+  entrypoint: /usr/local/bin/xe-daemon
+  environment:
+  - PATH=/sbin:/usr/local/bin
+  security:
+    writeableRootfs: false
+    writeableSysfs: true
+  mounts:
+  # libs
+  - source: /lib
```

Comment:
what's using `/lib`?

Reply:
@casdr is this PR good to go?

Reply:
@frezbo sorry for the late response, xe-daemon uses /lib/modules. I'll make this mount more explicit.

Reply:
I also guess this need a proper Xen host to test?

Reply:
@frezbo I don't think there's another way to test this unfortunately.

Reply:
no worries, I will exclude this from the talos extensions test suite